### PR TITLE
[Badges] Add Weekend Exerciser

### PR DESCRIPTION
**Pull request by iHiD**
Reopened https://github.com/exercism/v3/pull/1744

(cc @ynfle @kotp)

### CODE REVIEW

**Comment by ynfle** [comment]
Thanks! @iHiD

File: badges.md [hunk 1]
```diff
@@ -24,6 +24,7 @@ These will be in the codebase. They're unordered.
 - **Lackadaisical:** Solve bob in five languages
 - **Team builder:** Create a team with 3 members
 - **Teammate:** Give team feedback to 5 solutions.
+- **Weekend Exerciser:** Only submit solutions on the weekends for a month
```
Comment (SaschaMann) [inline badges.md:27]: [Different parts of the world have different weekends](https://en.wikipedia.org/wiki/Workweek_and_weekend#Around_the_world). Will the badge be localised or is there an implicit assumption of a "most of the world" Sat/Sun weekend?

Comment (SaschaMann) [inline badges.md:27]: [Different parts of the world have different weekends](https://en.wikipedia.org/wiki/Workweek_and_weekend#Around_the_world). Will the badge be localised or is there an implicit assumption of a "most of the world" Sat/Sun weekend?

Comment (ynfle) [reply badges.md:27]: Sounds like a considerable amount of effort for not much gain

Comment (iHiD) [reply badges.md:27]: Maybe "Wednesday Warrior" instead?

Comment (iHiD) [reply badges.md:27]:
```suggestion
- **Wednesday Warrior:** Only submit solutions on Wednesdays for 4 consecutive weeks
```

**Comment by iHiD** [comment]
So, I'll merge this on two conditions:
- A new category is created with the heading "May be implemented by the community" with the text "These badges will be implementable by PRs from the community nearer the time of v3 launch. We will announce when the repo is open for those PRs". Pop it under the two existing implement categories.
- This is moved to that category :)

**Comment by ynfle** [comment]
> So, I'll merge this on two conditions:
> 
> * A new category is created with the heading "May be implemented by the community" with the text "These badges will be implementable by PRs from the community nearer the time of v3 launch. We will announce when the repo is open for those PRs". Pop it under the two existing implement categories.
> * This is moved to that category :)

Just be clear, which version is being referred to?

**Comment by iHiD** [comment]
@ynfle Sorry. I mean the suggestion I put ("Wednesday Warrior"). (As apparently weekends aren't a "thing" as I'd previously thought)

**Comment by ErikSchierboom** [comment]
@iHiD Is there anything I can do to help with this PR?

**Comment by ErikSchierboom** [comment]
This Friday (Jan 29th 2021), we're going to explode this repo back into the individual track repositories and archive this repo.

As such, this PR needs to be merged by Friday 9am UTC, or it will need to be manually reopened against the track repo.

**Comment by iHiD** [comment]
> @iHiD Is there anything I can do to help with this PR?

@ErikSchierboom Yes please. Fulfil https://github.com/exercism/v3/pull/1752#issuecomment-649730854 and then merge 💙

**Review by ErikSchierboom — APPROVED** [review]

**Comment by iHiD** [comment]
Thanks :)

**Pull request merged**